LICENSE

```diff
@@ -1,6 +1,5 @@
 MIT License
 
-Copyright (c) 2023 redpangilinan
 Modifications Copyright (c) 2024 Turnkey
```

Comment:
Should "modifications" be removed as well?

Reply:
Duh!